SeleniumCookies/wrapper.py

```diff
@@ -308,7 +315,7 @@ def __add_session_cookies(self, cj):
             json_data = json.loads(
                 open(self.session_file, 'rb').read().decode())
         except ValueError as e:
-            print('Error parsing firefox session JSON:', str(e))
+            logging.error(f'Error parsing firefox session JSON: {str(e)}')

```

Comment:
Use error codes and predefined messages for handling exceptions

Reply:
Not getting the idea for error codes? Does this mean like JSON? 404 NOT FOUND? 
Taking exception in e and logging it, it's not predefined message handling exception?

Reply:
Using BrowserCookieError for handling all exceptions that occur for some reason like file not found or failed to open this etc. So in logging their reason is logged.

Reply:
No, custom defined error codes. These error codes should be mapped to a particular error message. For instance:

```python
CodeRejected = 1000
CodeInterrupted = 1001

ERRORS = []
ERRORS[CodeRejected] = "Rejection Exception"
ERRORS[CodeInterrupted] = "Interrupt Exception"
```

So whenever you print an error/warning, just pass the following details in the logger:

* ErrCode
* ERRORS[ErrCode]

Reply:
@L04DB4L4NC3R sorry for deletion, I didn't got your new message before deleting it.
Deleted because I strarted working with json codes.

Reply:
> @L04DB4L4NC3R sorry for deletion, I didn't got your new message before deleting it.
> Deleted because I strarted working with json codes.

Deletion of what?

Reply:
@L04DB4L4NC3R Deletion of my older query for which you have given explaination.

Reply:
@L04DB4L4NC3R PR maded